packages/manager/modules/pci/src/projects/project/ai-dashboard/tabs/users-tokens/users-tokens.html

```diff
@@ -0,0 +1,150 @@
+    <p data-translate="pci_ai_dashboard_users_tokens_description"></p>
+    <!--Users-->
+    <section>
+        <h4 data-translate="pci_ai_dashboard_users_tokens_users_title"></h4>
+        <div class="mb-3">
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_1"
+            ></span>
+            <br />
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_users_description_2"
+            ></span>
+        </div>
+
+        <div class="mb-3">
+            <oui-button
+                class="mr-2"
+                data-variant="secondary"
+                data-on-click="$ctrl.trackAndGoToManageUsers()"
+            >
+                <span
+                    data-translate="pci_ai_dashboard_users_tokens_users_manage_button"
+                    data-translate-values="{nbUsers: $ctrl.aiUsers.length}"
+                ></span>
+            </oui-button>
+            <oui-button
+                data-variant="primary"
+                data-on-click="$ctrl.trackAndGoToCreateUser()"
+            >
+                <i class="oui-icon oui-icon-add pr-1" aria-hidden="true"></i>
+                <span
+                    data-translate="pci_ai_dashboard_users_tokens_users_create_user_button"
+                ></span>
+            </oui-button>
+        </div>
+    </section>
+
+    <!--Tokens -->
+    <section>
+        <h4 data-translate="pci_ai_dashboard_users_tokens_tokens_title"></h4>
+        <div class="mb-3">
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_tokens_description_1"
+            ></span>
+            <br />
+            <span
+                data-translate="pci_ai_dashboard_users_tokens_tokens_description_2"
+            ></span>
+        </div>
+        <oui-datagrid data-page-size="10" data-rows="$ctrl.aiTokens">
+            <oui-datagrid-column
+                data-title=":: 'pci_ai_dashboard_users_tokens_tokens_table_name' | translate"
+                data-property="spec.name"
+                data-type="string"
+                data-searchable
+                data-filterable
+                data-sortable="asc"
+            ></oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'pci_ai_dashboard_users_tokens_tokens_table_labels_selector' | translate"
+                property="spec.labelSelector"
+                data-type="string"
+                data-sortable="asc"
+                data-searchable
+                data-filterable
+            ></oui-datagrid-column>
+            <oui-datagrid-column
+                data-title=":: 'pci_ai_dashboard_users_tokens_tokens_table_region' | translate"
+                data-property="spec.region"
+                data-type="string"
+                data-searchable
+                data-filterable
+                data-sortable="asc"
+            >
+                <span data-ng-bind="$row.spec.region"></span>
```

Comment:
**nitpick:** Its redundant. Can be removed.
```suggestion
```

Reply:
Yep, removed here: https://github.com/ovh/manager/pull/8443/commits/1e785e41932cdb7bbac7de9f5126103c8cb4d899